multi_company_partner/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Multi Company Partner
+=====================
+
+This basic module aims to allow one customer to be shared among
+    some of the sibling companies.
```

Comment:
s/to be shared among.../to be shared among several companies, not all or only one.